### PR TITLE
First stage

### PR DESCRIPTION
**Pull request by tatevikb**

### CODE REVIEW

File: thread-random/src/Counter.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        this.id = id;
+    }
+
+    @Override
+    public void run()
+    {
+        try {
+            int ms = new Random().nextInt(6) + 5;
+            Thread.sleep(ms * 1000);
+            synchronized (this) {
```
Comment (hhov88) [inline thread-random/src/Counter.java]: @tatevikb Move increment block to independent method

File: thread-random/src/Counter.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    public Counter(int id)
+    {
+        this.id = id;
+    }
+
+    @Override
+    public void run()
+    {
+        try {
+            int ms = new Random().nextInt(6) + 5;
```
Comment (hhov88) [inline thread-random/src/Counter.java:26]: @tatevikb does you thread work only one time ?

File: thread-random/src/Counter.java (hunk 1 above)
Comment (tatevikb) [reply thread-random/src/Counter.java]: Done

File: thread-random/src/Counter.java (hunk 2 above)
Comment (tatevikb) [reply thread-random/src/Counter.java:26]: Yes, each thread was working only once. I've made changes, now it works several times until the end of 1 minute.

Comment (tatevikb) [reply thread-random/src/Counter.java:26]: @hhov88

**Pull request merged**